app/assets/stylesheets/main.scss

```diff
@@ -7,7 +7,6 @@
 @import 'partials/events';
 @import 'partials/social_media';
 @import 'partials/chapter';
-@import 'partials/member_note';
```

Comment:
https://github.com/codebar/planner/pull/1455 changed an admin’s view of a user. Since then, I believe this CSS hasn’t been used (as well as the `app/assets/stylesheets/partials/_member_note.scss` partial below).